src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java

```diff
@@ -335,7 +335,7 @@ public void testChangeInConfig() throws Exception {
      * until #11589
      *
      * @noinspection ResultOfMethodCallIgnored
-     * @noinspectionreason ResultOfMethodCallIgnored - temporary suppression until #11589
+     * @noinspectionreason ResultOfMethodCallIgnored - Setup for mockito for which method to mock.
```

Comment:
can we be specific , something like `toByteArray only mocked to throw exception`